Adjust Play page for keywords (#3912)
camunda/camunda-docs: docs/components/modeler/web-modeler/play-your-process.md

- In **Implement** mode, hard-code an example payload in the task or event **Output** section.
- When completing a task or event, use the secondary action to complete it with variables.

> Incidents are raised as they would in any Camunda cluster, just within the context of Play, instead of Operate. Use the variables and incident message to debug the issue, all while staying in Play.

Reviewer: ```suggestion
Incidents are raised in Play just like in Operate. Use the variables and incident message to debug the process instance.
```

![rewind process](img/play-rewind.png)

> The rewind operation in Play currently does not support the following elements:

Reviewer: ```suggestion
Play's rewind operation currently does not support the following elements:
```
